content-services/content-sources-frontend
# Fixes 2938: Add Errata tab to snapshot details

## Summary

- Adds a query to retrieve snapshot errata
- Adds an Errata tab to SnapshotDetails modal to list snapshot errata
- Supports filtering by name / synopsis, type, and severity
- Dependent on these PRs: [backend](https://github.com/content-services/content-sources-backend/pull/608), [tang](https://github.com/content-services/tang/pull/6)

## Testing steps

- Add a repository with package errata and let it snapshot 
- Click View all snapshots from the kebab on the Repositories page
- Click the Errata count to route to the Errata (Advisories) tab in the SnapshotDetails modal
- You should see all the errata for this snapshot displayed, with additional info for each errata displayed when clicking the expandable icon 
- Filtering on name / synopsis, type, severity should only display errata with those filters
